docs/sources/trackers/google-tag-manager/snowplow-template/tag-types/index.md

```diff
@@ -0,0 +1,327 @@
+
+</details>
+
+### Timing
+
+This is used to track user timing events, such as how long resources take to load.
+
+<details plain>
+<summary>Timing Parameters</summary>
+
+| Name       | Required? | Description                    | Example            |
+| ---------- | --------- | ------------------------------ | ------------------ |
+| `category` | Yes       | Timing category                | 'load'             |
+| `variable` | Yes       | Timed variable                 | 'map_loaded'       |
+| `timing`   | Yes       | Number of milliseconds elapsed | 50                 |
+| `label`    | No        | Label for the event            | 'Map loading time' |
+
+</details>
+
+### Enhanced Consent
+
+The Enhanced Consent tag is used to track user consent preferences. This can be used to track whether a user has given consent to certain tracking activities, such as tracking cookies or personalized ads.
+
+<details plain>
+<summary>Enhanced Consent Events</summary>
+
+
+| Name           | Description                                             |
+| -------------- | ------------------------------------------------------- |
+| Allow All      | Track an acceptance of user consent.                    |
+| Allow Selected | Track a specific selection of consented scopes.         |
+| Pending        | Track the unconfirmed selection about user consent.     |
+| Implicit       | Track the implicit consent on user consent preferences. |
+| Deny           | Track a denial of user consent.                         |
+| Expired        | Track the expiration of a consent selection.            |
+| Withdrawn      | Track the withdrawal of user consent.                   |
+
+
+</details>
+
+
+<details plain>
+<summary>Enhanced Consent Parameters</summary>
+
+| Attribute            | Description                                                                         | Example                 |
+| -------------------- | ----------------------------------------------------------------------------------- | ----------------------- |
+| `basisForProcessing` | GDPR lawful basis for data collection & processing.                                 | Consent                 |
+| `consentUrl`         | URI of the privacy policy related document.                                         | https://example.com     |
+| `consentVersion`     | Version of the privacy policy related document.                                     | 1.0                     |
+| `consentScopes`      | The scopes allowed after the user finalized their selection of consent preferences. | 'analytics, functional' |
+| `domainsApplied`     | The domains for which this consent allows these preferences to persist to.          | https://example.com     |
+| `gdprApplies`        | Determine if GDPR applies based on the user's geo-location.                         | -                       |
+
+</details>
+
+
+### Enhanced Ecommerce
```

Comment:
why Enhanced Ecommerce in twice? also make it clear that this one is sort-of deprecated

Reply:
I've combined them into one, and added a little note about being deprecated.